instance/tests/integration/integration_instance.py

```diff
@@ -345,6 +343,9 @@ def test_spawn_appserver(self):
                 'static_template_about_content': 'Hello world!',
                 'homepage_overlay_html': '<h1>Welcome to the LMS!</h1>',
             },
+            s3_access_key='test-s3-access-key',
+            s3_secret_access_key='test-s3-secret-access-key',
+            s3_bucket_name='test-s3-bucket-name',
```

Comment:
This is an integration test that should be using real AWS credentials (which are already set as environment variables in CircleCI), so I don't think we should be setting these s3 fields here.
I think that's why the integration test is currently failing.

Reply:
Alright, thanks for telling me. However, I am having the `503` issue.